$NetBSD: patch-src_CMakeLists.txt,v 1.1 2024/07/08 14:04:42 micha Exp $

Remove" -Werror".
Add "-Wno-cast-function-type".
Remove duplicated flags (GCC fails because they are inserted with quotes).

--- src/CMakeLists.txt.orig	2024-07-04 06:00:57.000000000 +0000
+++ src/CMakeLists.txt
@@ -359,7 +359,7 @@ if(MSVC)
 endif()
 
 if ( CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID STREQUAL "GNU" )
-    target_compile_options(eurekasrc PUBLIC "${CMAKE_CXX_FLAGS}" -Wall -Wextra -Werror
+    target_compile_options(eurekasrc PUBLIC -Wall -Wextra -Wno-cast-function-type
                            -Wno-unused-parameter -Wno-missing-field-initializers -Wunused-variable)
 endif()
 if ( MSVC )
